common/http_test.cpp

```diff
@@ -1825,8 +1825,6 @@ TEST(HttpsTest, WrongSelfSignedCertificateError) {
 }
 
 TEST(HttpsTest, CertificationWithWrongHostName) {
-	GTEST_SKIP() << "MEN-6788";
```

Comment:
There are more like this in the proxy tests (grep for it).

Reply:
Oh, in a different file! :+1: